src/main/java/structures/Arrayer.java

```diff
@@ -21,7 +38,14 @@ public static int[] evensOnly(int[] input) {
      * ex: [1004, 9181, 1700, 4565] -> [4, 1, 0, 5]
      */
     public static int[] lastOfFourDigits(int[] input) {
-        return new int[0];
+        int[] result = new int[input.length];
+        // the result string will be the same length of the original string because we're simply editing the original string 
```

Comment:
good comment, but it's rather long. arguably should be broken up into two lines or multi-line. no points deduction this time.